### ISSUE TITLE
Yoshi Woolly World 60fps hub world?

### ISSUE THREAD

**Issue by JOHN30011887** [slashiee/cemu_graphic_packs#401]
Is it possible to have 60fps again in the main hub world?

It used to be 60fps and ran nice and smooth but then later cemu updates it locked it at 30fps on the hub world and switches to 60fps in the levels

Playing at 30 makes everything feel so sluggish and is really bad on the eye :(
I saw a few 60fps patches for cemu so thought id ask if possible for this game

**Comment by Xalphenos** [comment]
I could have sworn Theboy181 did this already.  Couldn't find a pack for it though so I did it.  I've only tested using the updated US version.  If anyone has the EU version to test it would be great!  If it doesn't work it'll be trivial to port it to other regions.

One thing to note though.  Before cemu implemented gx2setswapinterval, pretty sure that's the one, this game would run 60 on the world map but it was 2x game speed.  With this patch it runs 60 fps on the world map but maintains 1x game speed.
